README_TO_RUN_WITH_DREMAPLACE

```diff
@@ -0,0 +1,7 @@
+1. Please follow the link and install/build DREAMPlace
+    https://github.com/limbo018/DREAMPlace
+2. Build OpenROAD following the README.md 
```

Comment:
Should we fork the repo? Also, the install process can be included in build_openroad.sh and made optional same as LSOracle.

Reply:
I have to check with Professor Kahng and Ravi regarding this.

Reply:
Hi Vitor,
I checked with Prof. Kahng. He wants a sharable working version.
So I think we can skip the part about adding the DREAMPlace installer here.

Reply:
Adding to `build_openroad.sh` would make sharing easier in my opinion. I am also not a big fan of having this README_TO_DO_X... We should add this to the current README and the readthedocs as I have mentioned in my other comment below.

Reply:
I am not aware of how to add the installation details in build_openraod.sh. Also, it required some specific python environment to build it. So. I want to skip it. 
Regarding the extra README_TO_... file: is it okay to add all these details in the main README.md?